js/ngl/parser.js

```diff
@@ -2782,8 +2782,8 @@ NGL.CifParser.prototype = NGL.createObject(
 
             var symmetry = cif.symmetry;
 
-            var sGroup = symmetry[ "space_group_name_H-M" ];
-            if( sGroup[0] === sGroup[ sGroup.length-1 ] &&
+            var sGroup = symmetry[ "space_group_name_H_M" ];
```

Comment:
The field in mmCIF has a hyphen between H and M i.e. `space_group_H-M`. Does it work with underscore?